From 1e39e6b296b94852e73adea1cb652adda435529e Mon Sep 17 00:00:00 2001
From: Anton Kudryavtsev <a.kudryavtsev@netris.ru>
Date: Mon, 21 Mar 2016 16:43:43 +0300
Subject: [PATCH] assimp: Fix build FBXConverter.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

If you compile according to https://wiki.qt.io/Qt5ForAndroidBuilding
with NDK v11.b, you get such warning and the compilation stops:

FBXConverter.cpp:1262:19: warning: ‘count’ may be used uninitialized in this function [-Wmaybe-uninitialized]
      unsigned int count;

Change-Id: Ieff61e6302800a22080f837267982fd2ea3129e6
---
 src/3rdparty/assimp/code/FBXConverter.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/3rdparty/assimp/code/FBXConverter.cpp b/src/3rdparty/assimp/code/FBXConverter.cpp
index 56643bc..c6ef8c9 100644
--- a/src/3rdparty/assimp/code/FBXConverter.cpp
+++ b/src/3rdparty/assimp/code/FBXConverter.cpp
@@ -1259,7 +1259,7 @@ private:
 				// taking notes so we don't need to do it twice.
 				BOOST_FOREACH(WeightIndexArray::value_type index, indices) {
 
-					unsigned int count;
+					unsigned int count = 0;
 					const unsigned int* const out_idx = geo.ToOutputVertexIndex(index, count);
 
 					index_out_indices.push_back(no_index_sentinel);
-- 
1.9.1

